src/js/main.js

```diff
@@ -1,6 +1,18 @@
+/* This is a demo of how you can import another JS file */
 import { printImportWorks } from "./other-file";
+/* You can even import other resources this way */
 import image from '../assets/images/image.jpg';
 
 console.log('main js file works');
-console.log(' here is the path for a asset in js', image)
-printImportWorks()
+console.log('here is the path for a asset in js', image);
+printImportWorks();
+
+if ('serviceWorker' in navigator) {
+	navigator.serviceWorker.register('../service-worker.js').then(function (registration) {
```

Comment:
as this aims to be for beginers and only for playgrounds,  not sure if service workers should be added

Reply:
OK, but this only adds the static files in a local cache, it's a good thing to have. We can remove it if it doesn't make sense, but it helped me rank better in Lighthouse for several previous projects...